index.bs

```diff
@@ -1018,6 +1015,11 @@ Composition of the Technical Architecture Group</h5>
 	and <em class=rfc2119>may</em> be run at other times when initiated by the current chairs or the Team,
 	for example if a chair steps down or if a minority of the participants make such a request.
 
+	As <dfn export>Director Emeritus</dfn>,
+	Tim Berners-Lee has an ongoing standing invitation
```

Comment:
There's no expectation that this ever will be plural (and if it did become plural, that would imply a change in governance large enough to merit thinking about explicitely), so I think it makes sense to keep this in the singular.

Reply:
As for using this as a dfn vs a reference to a dfn, "Tim Berners Lee" _is_ effectively the definition of "Director Emeritus" for our purposes, so I think the original phrasing is fine, and actually, removing his name from this definition suggests, incorrectly, that it might apply to more people, which is not intended.

Reply:
```suggestion
	As the only <dfn export>Director Emeritus</dfn>,
	Tim Berners-Lee has an ongoing standing invitation
```

Reply:
I wonder if we should drop line 1018 entirely. While timbl was *the* Director for the longest time, that word has now taken a new meaning (board member). The "the only" clarification helps, but maybe dropping the line altogether is simpler. Or maybe replace "Director Emeritus" with "founder of W3C", which unambiguously refers to him and him alone.

Reply:
```suggestion
	As <dfn export>founder of W3C</dfn>,
	Tim Berners-Lee has an ongoing standing invitation
```

Reply:
Is this missing some articles?

"As the founder of the W3C," or "As founder of the W3C," both work for me.  I realize that the usage of the standalone "W3C" in this document often deliberately omits a "the", but that just reads as broken English to this native speaker.

Reply:
Omitting "the" in front of "W3C" is not limited to this document, it's W3C's official branding guidelines, which we merely follow. Took me a while to get used to, but I've now grown accustomed to it, and it doesn't really strike me as any more strange than the lack of article in front of IBM or NASA.